src/components/Pills/pills.module.scss

```diff
@@ -3,8 +3,8 @@
   --label: var(--blue-color);
   --hover-bg: var(--blue-color-30);
   --hover-label: var(--text-primary-color);
-  padding: $space-xxs $space-s;
-  border-radius: $border-radius-m;
+  padding: 5px $space-xs;
+  border-radius: 6px;
```

Comment:
do we want to expose as a css var? probably not 🤷

Reply:
Not for padding, but maybe we _should_ expose the border radius.